packages/components/.storybook/preview.tsx

```diff
@@ -1,8 +1,10 @@
+import "@chanzuckerberg/czedi-kit-tokens/css/variables.css";
+
 import * as React from "react";
 
 import { DocsContainer, DocsPage } from "@storybook/addon-docs/blocks";
-import EDSGlobalStyles from "../src/styles/global";
 
+import EDSGlobalStyles from "../src/styles/global";
```

Comment:
I think this is probably a result of my auto formatting, I'll add a rule to our eslint so it applies for all of us.